### PR TITLE
Missing changelog message regarding Logback environment variables.  Closes #345

### LINKED ISSUE #345: LOG_LEVEL support in run command
Hi.  Thanks for a great project.  Attempting to bring up docker container and hit REST endpoints.
Looks like I needed to add -DLOG_LEVEL to `run` command

```
# pwd
/etc/service/cromwell
# cat run
#!/bin/bash
exec java $JAVA_OPTS -DLOG_LEVEL=$LOG_LEVEL -Djava.library.path=./native -jar /cromwell/cromwell-0.14.jar server
```

### PR DESCRIPTION
**Pull request by kshakir**
Also closes #345.

### CODE REVIEW

**Comment by mcovarr** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ task disk_test {
 ```
 * Added workflow options to copy the call logs and/or the workflow logs to a `call_logs_dir` or a `workflow_log_dir`,
 respectively.
+* The system properties `LOG_MODE` and `LOG_LOGLEVEL` used by Logback may now be specified as environment variables.
```
Comment (cjllanwarne) [inline CHANGELOG.md]: Is this also in the README?

**Comment by cjllanwarne** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (scottfrazer) [inline CHANGELOG.md]: `LOG_LOGLEVEL`?  is that right?  not `LOG_LEVEL`?

**Comment by kshakir** [comment]
Typos fixed and mentioned to the README. Will merge in a few minutes unless there are other comments.

**Comment by scottfrazer** [comment]
:+1:

**Pull request merged**